src/cases/caseManager.ts

```diff
@@ -44,6 +44,14 @@ export class CaseManager {
 		return this.#cases.values();
 	}
 
+	public getCase(caseHash: CaseHash): Case | undefined {
```

Comment:
I would say `Case | null` fits here better (`this.#cases` cannot have `null` values for any key either way)